articles/extensions/delegated-admin.md

```diff
@@ -0,0 +1,148 @@
+
+- **EXTENSION_CLIENT_ID**: The **Client ID** value of the Client you will use.
+- **TITLE**: Optionally, you can set a title for your Client. It will be displayed at the header of the page.
+- **CUSTOM_CSS**: Optionally, you can set a css file, to customize the look and feel of your Client.
+
+Once you have provided this information, click **Install**. Your extension is now ready to use!
+
+
+## Use the extension
+
+To access your newly created dashboard, navigate to *[Extensions](${uiURL}/#/extensions) > Installed Extensions* and click on the **Delegated Administration Dashboard**. A new tab will open and display the login prompt. 
+
+![](/media/articles/extensions/delegated-admin/login-prompt.png)
+
+Notice that there is no Sign Up option. That's because we disabled it earlier. 
+
+Once you provide valid credentials you are navigated to the *Delegated Administration Dashboard*.
+
+![](/media/articles/extensions/delegated-admin/standard-dashboard.png)
+
+There are two available views, *Users* and *Logs*. At the *Users* view you can see all the users displayed and perform certain actions on them. In the table below you can see all the options you can perform on a user, which ones are available via the [dashboard](${uiURL}/#/) and which via the extension. 
+
+<table class="table">
+    <tr>
+        <th>Action</th>
+        <th> Available via Dashboard </th>
+        <th> Available via Extension </th>
+    </tr>
+    <tr>
+        <th>Create user</th>
+        <th>Yes</th>
+        <th>No</th>
+    </tr>
+    <tr>
+        <th>Contact user</th>
+        <th>Yes</th>
+        <th>No</th>
+    </tr>
+    <tr>
+        <th>Sign in as user</th>
+        <th>Yes</th>
+        <th>No</th>
+    </tr>
+    <tr>
+        <th>Block user</th>
+        <th>Yes</th>
+        <th>Yes</th>
+    </tr>
+    <tr>
+        <th>Delete user</th>
+        <th>Yes</th>
+        <th>Yes</th>
+    </tr>
+    <tr>
+        <th>Send verification email</th>
+        <th>Yes</th>
+        <th>No</th>
+    </tr>
+    <tr>
+        <th>Change email</th>
+        <th>Yes</th>
+        <th>No</th>
+    </tr>
+    <tr>
+        <th>Change password</th>
+        <th>Yes</th>
+        <th>Yes</th>
+    </tr>
+    <tr>
+        <th>Reset password</th>
+        <th>No</th>
+        <th>Yes</th>
+    </tr>
+</table>
+
+Notice the new *Reset password* option available via the extension. This option will send an email to the user allowing them to choose a new password. To do this click on a user and select *Actions > Reset password*.
+
+![](/media/articles/extensions/delegated-admin/reset-pass-01.png)
+
+This will send an email to the user, containing a link to change the password.
+
+At the *Logs* view you can see log data of authentications made by your users. The contents of this view are a subset of the data displayed in the [Logs dashboard](${uiURL}/#/logs), which also displays data for the actions taken in the dashboard by the administrators.
```

Comment:
@sandrinodimattia Is this section on the Logs view valid? I can see the `API Operation` logs only on my dashboard, they are not displayed via the extension.